md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ext/DataLoadAPI.java

```diff
@@ -89,8 +94,7 @@ RestWrapper list(Principal principal) {
 
     @RequestMapping(value = {"/createjobs"}, method = RequestMethod.POST)
 
-    public
-    @ResponseBody
+    @ResponseBody public
     RestWrapper createJob(@RequestParam Map<String, String> map, Principal principal) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "createJob" is 35 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)